### PR TITLE
Update styles

### LINKED ISSUE #4935: Change Setting Options' toggles to anathor colour
### Is your request related to a problem? Please describe.
Since there's the colour-changing on beta versions, I don't like that the toggles of the setting options are still blue. It can be comfusing, and doesn't look that much good. Also the images doesn't turn ito blue on prereleases, which looks kind of ugly.

### Describe the solution you'd like

Change the mentioned colours.
### Describe alternatives you've considered

None others yet.
### Additional context

Discord suggestion: https://discord.com/channels/806602307750985799/1003668150660907088
(Which became the first step of the story of my banning out of SA server(s). This is partially why I made GH issue as well.)

### PR DESCRIPTION
**Pull request by OregSamSas**
Resolves #4935

### Changes

I've replaced most of the orange colours when it's a prerelease version, and also removed the blue colour of the setting options' toggles. Now it's a less saturated type of that colour which is set to the main. (i. e. the "--brand-orange" variable)
   
<details>
<summary>Screenshot1</summary>

</details>
<details>
<summary>Screenshot2</summary>

</details>
<details>
<summary>Screenshot3</summary>

</details>
<details>
<summary>Screenshot4</summary>

</details>
<details>
<summary>Screenshots 5, 6</summary>

</details>
<details>
<summary>Screenshots 7, 8</summary>

</details>

### Reason for changes

Changing helps distinguish whether the enabled addon is a beta version or not, and also looks better. 
### Tests

I've tested all of the changes on all platforms (Chrome 103.0, Waterfox  G4.1.3.2, Edge 103.0), and sadly it didn't work somehow on Microsoft Edge with prerelease versions, but I open the pull request anyway (considering that not many people will probably use Edge for beta-testing)

### CODE REVIEW

File: webpages/styles/colors.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 :root {
   --brand-orange: #ff7b26;
   --brand-orange-opacity35: rgba(255, 123, 38, 0.35);
+  --brand-orange-option: rgba(255, 123, 38, 0.7);
```
Comment (Samq64) [inline webpages/styles/colors.css:4]: `brand-orange-option` isn't very clear, it should probably be named something like `brand-orange-opacity70`

**Comment by OregSamSas** [comment]
Wait there's #4934

**Comment by OregSamSas** [comment]
Borders on focusing in this PR changes as well like this:
image

**Comment by OregSamSas** [comment]
But why did this get closed? I don't get it.
I would appreciate if I got a little bit more details @apple502j, thanks in advance.

(My guesses would be because it has conflicts with that PR, or it's already implemented with another PR, or the changes wasn't necessary, but I haven't got any exact idea.)

**Comment by OregSamSas** [comment]
I can change the colour of focused input borders, if it's the reason.

**Pull request closed without merging**